homeassistant/components/zwave_js/sensor.py

```diff
@@ -468,6 +468,25 @@ async def async_reset_meter(
 class ZWaveListSensor(ZwaveSensor):
     """Representation of a Z-Wave Numeric sensor with multiple states."""
 
+    def __init__(
+        self,
+        config_entry: ConfigEntry,
+        driver: Driver,
+        info: ZwaveDiscoveryInfo,
+        entity_description: SensorEntityDescription,
+        unit_of_measurement: str | None = None,
+    ) -> None:
+        """Initialize a ZWaveListSensor entity."""
+        super().__init__(
+            config_entry, driver, info, entity_description, unit_of_measurement
+        )
+
+        # Entity class attributes
+        self._attr_name = self.generate_name(
+            alternate_value_name=self.info.primary_value.property_name,
```

Comment:
Is there a finite list of known properties according to the spec or is this something that each device can make up? Long term I'd like to move to using entity descriptions and if we know all the possible names, we could set the name there too.

If we have a reference about this, can we add that as a comment so it's easier to refactor in the future?

Reply:
done 👍🏾